styles/styles.css

```
@@ -0,0 +1,611 @@
+    text-align: center;
+}
+
+.catalog-background-header {
+    background-image: url(../images/newsletter-background.jpg);
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: center;
+    color: #FFFFFF;
+    background-color: #82B3D3;
+}
+
+.catalog-title {
+    font-weight: 700;
+    font-size: 60px;
+    line-height: 78px;
+}
+
+.breadcrumbs-hotels {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 21px;
+    color: #FFFFFF;
+}
+
+.catalog-legend{
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    text-transform: capitalize;
+}
+
+.infrastructure-list,
+.type-hotel {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 23px;
+}
+
+
+.about-sedona {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+    display: flex;
+    justify-content: center;
+    margin-top: 69px;
+    margin-left: 290px;
+    margin-bottom: 120px;
+    margin-right: 290px;
+    padding: 0;
+}
+
+.text-description {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    color: #333333;
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin-top: 25px;
+    margin-left: 274,5px;
+    margin-bottom: 69px;
+    margin-right: 274,5px;
+}
+
+
+.advantages-title {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+
+.reservations-title {
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+.reservations-text {
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 26px;
+    text-align: center;
+    color: #333333;
+    text-transform: none;
+}
+
+.reservations-button {
+    font-family: inherit;
+    text-transform: uppercase;
+}
+
+.visually-hidden {
+    position: fixed;
+    transform: scale(0);
+}
+
+.number-of-selected-hotels {
+    color:#FFFFFF;
+    background-color: #7DB54F;
+    text-decoration: none;
+    width: 20px;
+    height: 20px;
+    border-radius: 50%;
+    font-weight: 400px;
+    font-size: 10px;
+    line-height: 10px;
+    text-align: center;
+}
+
+.advantages-title-first,
+.advantages-title-fifth {
+    color:#FFFFFF;
+    text-align: center;
+}
+
+.advantages-description-first,
+.advantages-description-fifth {
+    color:#FFFFFF;
+    text-align: center;
+}
+
+.newsletter-text {
+    font-family: 'PT Sans';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 26px;
+    color: #FFFFFF;
+    text-align: center;
+    text-transform: none;
+}
```

```suggestion
.newsletter-text {
    font-weight: 400;
    font-size: 22px;
    line-height: 26px;
    color: #FFFFFF;
    text-align: center;
    text-transform: none;
}
```

Ну откуда же они вылезают??
Вот эти:
```
    font-family: 'PT Sans';
    font-style: normal;
```